addons/mail/static/src/js/systray/systray_activity_menu.js

```diff
@@ -111,13 +111,18 @@ var ActivityMenu = Widget.extend({
         if (actionXmlid) {
             this.do_action(actionXmlid);
         } else {
+            var domain = [['activity_ids.user_id', '=', session.uid]]
+            if (targetAction.data('domain') != null) {
```

Comment:
I think  if (targetAction.data('domain') {} is better has we should probably not take into account domains being False, [], ...

Reply:
Updated it!